### DISCUSSION TITLE
Foster email requests

### DISCUSSION THREAD

**Discussion by felinerescue** [sheltermanager/asm3#1436] [General]
Hello - 

Is there a good way to quickly pull information about animals and email it to our fosters directly out of ASM? Our current process for requesting foster placements is very tedious and time-consuming. It would be ideal if we could email out information from a template that auto-populates from the cats' files we need foster placement for. Any suggestions on how to make this happen?

Thanks!

**Comment by Tetzor** [comment]
Hi,

If you create and add a "needs foster" flag to these animals, you could then use the html_flagged_animals service call to find them.

You could make a HTML template to use in the call that includes all the information you want from the animal records. Then you could put it up as a page on your website and send a link to it in your email to fosterers, or just send them the service call to click on. I have added a link to the manual below to help you. 

https://sheltermanager.com/repo/asm3_help/service.html#html-flagged-animals

Jon
